jupyter_alabaster_theme/static/pcss/mediaqueries.pcss

```diff
@@ -158,6 +158,13 @@
       display: none;
     }
 }
+/* Medium sized devices */
+@media (max-width: 768px) {
```

Comment:
@cameronoelsen could you group this with the other `@media (max-width: 768px)` on line 100? After that run `npm run css` to update the CSS files with changes.